docs/ddl-of-carbondata.md

```diff
@@ -104,17 +104,18 @@ CarbonData DDL statements are documented here,which includes:
 
  Following are the guidelines for TBLPROPERTIES, CarbonData's additional table options can be set via carbon.properties.
 
-   - ##### Dictionary Encoding Configuration
+   - **Dictionary Encoding Configuration**
```

Comment:
These needs to be headings so that links can be created to this.Infact there are links to these

Reply:
handle these in all other places modified